openshift_cli_installer/libs/clusters/ocp_cluster.py

```diff
@@ -139,6 +144,19 @@ def _add_s3_bucket_data(self):
             f"{self.cluster_info['name']}-{self.cluster_info['shortuuid']}.zip"
         )
 
+    def check_and_assign_aws_cluster_region(self):
+        if not self.cluster_info.get("region"):
```

Comment:
why do you need to check `if not self.cluster_info.get("region")`? 
you call this function only when `if self.create and self.select_aws_auto_region:`